Mathlib/Geometry/Manifold/ContMDiff/Basic.lean

```diff
@@ -259,6 +259,55 @@ alias contMDiffAt_of_not_mem := contMDiffAt_of_notMem
 alias contMDiffAt_of_not_mem_mulTSupport := contMDiffAt_of_notMem_mulTSupport
 
 
+/-! ### Being `C^k` on a set can be tested in its union -/
+section contMDiff_union
+
+variable {s t : Set M}
+
+/-- If a function is `C^k` on two open sets, it is also `C^n` on their union. -/
+lemma ContMDiffOn.union_of_isOpen (hf : ContMDiffOn I I' n f s) (hf' : ContMDiffOn I I' n f t)
```

Comment:
Question for all of those: is `_of_isOpen` needed? Are there other true lemmas you'd confuse them with if you removed the suffix?